modules/ROOT/pages/Development/TestingResources.adoc

```diff
@@ -149,6 +149,14 @@ $Args2 = "-EpicPortal", "-NoSteamClient", '-Username="'+$Username2+'"', "-WinX=9
 
 == Multiplayer Testing
 
+[WARNING]
+====
+Update 8 has recently changed how local multiplayer testing works.
+This page has not yet been updated to reflect these changes.
+You must load with the `-offline` flag and use a Map Travel URL with the `?listen` argument to be able to connect.
+For example, `open Persistent_Level?loadgame=NameOfYourSaveGame?listen`.
```

Comment:
I think this will mean SessionSettings are discarded

Reply:
Correct, this has always been the case with the launch script and session settings. I'll add a note about it though.